config/hsc/ingest.py

```diff
@@ -0,0 +1,8 @@
+from lsst.obs.subaru.ingest import HscParseTask
```

Comment:
I assume that the user needs to know to pass `--configfile $OBS_SUBARU_DIR/config/hsc/ingest.py`.  Is that true?  If so, can you make sure it's documented and if not, how does the `CmdLineTask` find the configs within a subdirectory?

Reply:
Our `ArgumentParser` will load `config/<taskName>.py` and `config/<instrumentName>/<taskName>.py` automatically.